src/traces/parcoords/shaders/pickVertex.glsl

```diff
@@ -0,0 +1,82 @@
+precision highp float;
```

Comment:
@monfera I just noticed this. Can you rename this file `pick_vertex.glsl`? We prefer using lower case file name as some file systems are case insensitive. 

We actually have a [test](https://github.com/plotly/plotly.js/blob/f5239521ffea03b3b4f3ebad995663396f203a32/tasks/test_syntax.js#L97-L103) checking for that, but it checks `*.js` files at the moment :disappointed: